website/community/2-resources.mdx

```diff
@@ -34,36 +34,36 @@ See the <a href={require('@docusaurus/useBaseUrl').default('showcase')}>showcase
 
 ### Search {#search}
 
-- [docusaurus-plugin-lunr](https://github.com/daldridge/docusaurus-plugin-lunr) - Docusaurus v2 plugin to create a local search index for use with Lunr.js
-- [docusaurus-lunr-search](https://github.com/lelouch77/docusaurus-lunr-search) - Offline Search for Docusaurus v2
-- [docusaurus-search-local](https://github.com/cmfcmf/docusaurus-search-local) - Offline/local search for Docusaurus v2
-- [@easyops-cn/docusaurus-search-local](https://github.com/easyops-cn/docusaurus-search-local) - Offline/local search for Docusaurus v2 (language of zh supported)
-- [docusaurus-theme-search-typesense](https://github.com/typesense/docusaurus-theme-search-typesense) - Docusaurus v2 plugin for [Typesense DocSearch](https://typesense.org/docs/latest/guide/docsearch.html)
+- [docusaurus-plugin-lunr](https://github.com/daldridge/docusaurus-plugin-lunr) - Docusaurus plugin to create a local search index for use with Lunr.js
+- [docusaurus-lunr-search](https://github.com/lelouch77/docusaurus-lunr-search) - Offline Search for Docusaurus
+- [docusaurus-search-local](https://github.com/cmfcmf/docusaurus-search-local) - Offline/local search for Docusaurus
+- [@easyops-cn/docusaurus-search-local](https://github.com/easyops-cn/docusaurus-search-local) - Offline/local search for Docusaurus (language of zh supported)
+- [docusaurus-theme-search-typesense](https://github.com/typesense/docusaurus-theme-search-typesense) - Docusaurus plugin for [Typesense DocSearch](https://typesense.org/docs/latest/guide/docsearch.html)
 - [@markprompt/docusaurus-theme-search](https://github.com/motifland/markprompt-js/tree/main/packages/docusaurus-theme-search) - Docusaurus plugin to add generative AI / LLM-powered docs search, powered by [Markprompt](https://markprompt.com).
 
 ### Integrations {#integrations}
 
-- [docusaurus2-dotenv](https://github.com/jonnynabors/docusaurus2-dotenv) - A Docusaurus 2 plugin that supports dotenv and other environment variables
-- [posthog-docusaurus](https://github.com/PostHog/posthog-docusaurus) - Integrate [PostHog](https://posthog.com/) product analytics with Docusaurus v2
+- [docusaurus2-dotenv](https://github.com/jonnynabors/docusaurus2-dotenv) - A Docusaurus plugin that supports dotenv and other environment variables
```

Comment:
Care to take any action on plugins that are named after Docusaurus v2 with no hints that it will support Docusaurus v3 now or in future?

Could consider dropping them from the list, and someone can add it again when or if it has Docusaurus v3 support or someone makes an alternative that supports Docusaurus v3.

Not a big deal if they stay, but it can be a bit noisy/distracting, so thought it was brining attention to it.